src/app/collections/sed/sed.html

```diff
@@ -1,8 +1,11 @@
 <ai-collection call-function="vm.viewDetails(id)" collection-key="sed" columns="vm.columnSet" filters="vm.filters" refine-model="vm.refineModel">
   <ai-body-text>
-    <p>
+    <div>
       This list includes all health IT products that have been certified with Safety Enhanced Design (SED):
-    </p>
+      <div class="btn-group">
+        <a type="button" class="btn btn-sm btn-ai-success" href="{{ vm.API }}/certified_products/sed_details?api_key={{ vm.API_KEY }}"><i class="fa fa-cloud-download"></i><span class="sr-only">Download All SED Details</span>Download All SED Details</a>
+      </div>
```

Comment:
A screen-reader reading this would have "Download All SED Details" read out loud twice. The `class="sr-only"` is used to make text only parsed by screen-readers, but not shown visibly.